Library/Formula/kobalt.rb

```diff
@@ -0,0 +1,46 @@
+class Kobalt < Formula
+  desc "Build system"
+  homepage "http://beust.com/kobalt"
+  url "https://github.com/cbeust/kobalt/releases/download/0.633/kobalt-0.633.zip"
```

Comment:
Are you saying that when someone installs Kobalt, you will build it from the source and then install the resulting binaries? (sorry if it's a silly question, I'm not very familiar with Homebrew's internals)